Makefile

```
@@ -146,6 +146,10 @@ add-license-headers: $(GO_ADD_LICENSE)
 check-generate:
 	@hack/check-generate.sh $(REPO_ROOT)
 
+.PHONY: check-plutono-dashboards
+check-plutono-dashboards:
+	@./hack/check-plutono-dashboards.sh
```

```suggestion
	@hack/check-plutono-dashboards.sh
```

Just for consistency, a few lines below you use the `@hack...` form - that could be used here as well.